### ISSUE TITLE
BUG -  Unexpected charge to the swell account on cancelling a subscription

### ISSUE THREAD

**Issue by Daniel-Alamezie** [swellstores/swell-node#35]
I came across this catastrophic bug yesterday on swell, which I hope gets looked into soon.

I noticed that when you create a subscription manually on the swell dashboard and you then try to cancel/delete the subscription, it doesnt delete the subscription, but then charges your account for the initial yearly amount of your swell subscription.

Steps to reproduce
------------------------
- Create a subscription product
- Navigate to subscription tab
- Create a new subscription under the new subscription product
- Edit the subscription by trying to delete it
- 
![image](https://github.com/swellstores/swell-node/assets/74880857/8677de66-3072-4314-bb83-41f9ab837747)

- refresh the page
- 
![image](https://github.com/swellstores/swell-node/assets/74880857/929e20a8-7297-47d4-9651-5c95d19aa5dc)
This actually atempts a charge to my bank too on my actual account

**Comment by steven-k1** [comment]
@Daniel-Alamezie Looking into this

**Comment by Daniel-Alamezie** [comment]
Thank you

**Comment by steven-k1** [comment]
@Daniel-Alamezie I wasn't able to recreate the scenario you presented, but what I did find is that our system generated an invoice for your Swell subscription and was marked past due, which is the reason for the 'Account payment' notice that popped up in the dashboard.

I've voided that invoice, so now the notice should no longer show up, and we are looking into the reason for the invoice generation.

However, I can confirm that this is not connected to generating a subscription.
